scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -96,10 +96,9 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
                 .Execute()
                 .EnsureSuccessful();
 
-            // The 'ProjectWithTests' is a portable test app. Cannot call --infer-runtimes on it, since on win x64 machines,
-            // the x86 runtime is being inferred, and there are no x86 DotNetHost packages
             dotnet.Restore(
```

Comment:
This project can now be folded back into TestAssets\TestProjects.  The reason it was separate is because of the problem you are solving. Now there is no reason for it to be separate anymore.

Although it isn't critical for your work to do it, and may just make it longer.  So if you aren't going to do this, can you either open an issue, or put a TODO here?